vendor/engines/secure_rooms/spec/controllers/secure_rooms_api/scans_controller_spec.rb

```diff
@@ -7,11 +7,44 @@
       password = Settings.secure_rooms_api.basic_auth_password
       encoded_auth_credentials = ActionController::HttpAuthentication::Basic.encode_credentials(name, password)
       request.env['HTTP_AUTHORIZATION'] = encoded_auth_credentials
-      post :scan
+
+      post :scan,
+           card_id: card_number,
+           controller_id: control_device_id,
+           reader_id: card_reader_id
+    end
+
+    subject { response }
+
+    let(:card_reader) { create :card_reader }
+    let(:card_user) { create :user, card_number: '123456' }
+
+    let(:card_reader_id) { card_reader.id }
```

Comment:
I'm not really sure these `let`s are carrying their weight. I think you'd be fine if you used the dot version in the `post`.